tms/clickup/views.py

```diff
@@ -19,3 +23,15 @@ def post(self, request):
         return JsonResponse({'token': token.key,
                              "user": user.username,
                              "message": "Successfully logged in."}, status=status.HTTP_200_OK)
+
+
+class ProjectCreateView(ListCreateAPIView):
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+
+
+class ProjectRetrieveUpdateDestroyView(RetrieveUpdateDestroyAPIView):
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
Every user can create the project from this permission class it should be dynamic if I have he permission then only I can create